Classes/Controller/DocumentsController.php

```diff
@@ -63,6 +63,20 @@ class DocumentsController extends ActionController
      */
     protected $siteRepository;
 
+    /**
+     * @var array
+     */
+    protected $settings = [];
```

Comment:
We could directly inject the setting:

- `@Flow\InjectConfiguration(path="administrator.name")`